Source/WebCore/html/HTMLOptGroupElement.cpp

```diff
@@ -65,6 +65,12 @@ bool HTMLOptGroupElement::isFocusable() const
 {
     if (!supportsFocus())
         return false;
+
+    RefPtr select = ownerSelectElement();
+    if (select && select->usesMenuList())
+        return false;
+    return HTMLElement::isFocusable();
+
     // Optgroup elements do not have a renderer.
     auto* style = const_cast<HTMLOptGroupElement&>(*this).computedStyle();
     return style && style->display() != DisplayType::None;
```

Comment:
The code below is no longer executed

Reply:
Similar suggestion as above? Will it work?

Reply:
I think the whole function can probably be:
```
     RefPtr select = ownerSelectElement();
     if (select && select->usesMenuList())
         return false;
     return HTMLElement::isFocusable();
```

The display & the supportsFocus() checks are both covered by `HTMLElement::isFocusable()`.

@rniwa Wdyt?

Reply:
Removed. :-)